### PR TITLE
fix: related to #13 telescope-project, enter insert mode on prompt close

### PR DESCRIPTION
**Pull request by claidler**
@Conni2461

### CODE REVIEW

**Comment by Conni2461** [comment]
Your branch is 142 commits behind, so you already have to rebase. Sorry :)

**Comment by claidler** [comment]
@Conni2461 only 142. Okay, no worries!

File: lua/telescope/actions/init.lua [hunk 1]
```diff
@@ -196,8 +196,8 @@ function actions.close_pum(_)
   end
 end
 
-local do_close = function(prompt_bufnr, keepinsert)
-  local picker = action_state.get_current_picker(prompt_bufnr)
+actions._close = function(prompt_bufnr, keepinsert)
+  local picker = actions.get_current_picker(prompt_bufnr)
```
Comment (Conni2461) [inline lua/telescope/actions/init.lua:200]: This needs to be `action_state`

File: lua/telescope/actions/init.lua [hunk 2]
```diff
@@ -59,7 +59,7 @@ function actions.get_current_line()
   return action_state.get_current_line()
 end
 
-function actions.get_current_picker(prompt_bufnr)
+function action_state.get_current_picker(prompt_bufnr)
```
Comment (Conni2461) [inline lua/telescope/actions/init.lua:62]: This needs to be `actions`.

Otherwise thanks :)

Comment (claidler) [reply lua/telescope/actions/init.lua]: Thanks @Conni2461 I think this is resolved now :-)

File: lua/telescope/actions/init.lua (hunk 1 above)
Comment (Conni2461) [reply lua/telescope/actions/init.lua:200]: `actions.get_current_picker` is deprecated. So can we keep `action_state.get_current_picker` here.

@elianiva After this is fixed you can merge. It also closes some issue if i recall correctly. I can't/won't be here for 2 weeks. So i hand it off to you, if that is okay :) thanks <3

Comment (claidler) [reply lua/telescope/actions/init.lua]: @elianiva I think this should be good now

**Comment by elianiva** [comment]
Thank you! :)

**Comment by claidler** [comment]
Thanks @elianiva :-)

**Pull request merged**